x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -75,18 +76,19 @@ export function generateTemplateName(dataset: Dataset): string {
   return getDatasetAssetBaseName(dataset);
 }
 
-function getBaseTemplate(mappings: Mappings): Template {
+function getBaseTemplate(type: string, mappings: Mappings): Template {
   return {
     // We need to decide which order we use for the templates
     order: 1,
     // To be completed with the correct index patterns
     index_patterns: [],
     settings: {
       index: {
-        // ILM Policy must be added here
+        // ILM Policy must be added here, for now point to the default global ILM policy name
         lifecycle: {
-          name: 'logs-default',
-          rollover_alias: 'logs-nginx-access-abcd',
+          name: `${type}-default`,
+          // i don't think we will be using this once datastream/index templates v2 is supported
+          // rollover_alias: `logs-nginx-access-abcd`,
```

Comment:
@ruflin is it okay to remove this as it looks like we won't need to set this in the near future?  or do we need something for the interim?

Reply:
yes, please clean it up.